examples/openspending/components/Header.tsx

```diff
@@ -16,18 +16,22 @@ export function Header() {
   };
 
   const navLinks = [
-    {
-      title: 'Home',
-      href: '/',
-    },
     {
       title: 'Datasets',
       href: '/#datasets',
     },
-    // {
-    //   title: "Community",
-    //   href: "https://community.openspending.org/"
-    // }
+    {
```

Comment:
One task that's missing is to align the navbar items to the right

Reply:
ok, i thought you wouldnt want that after the navbar started having more stuff, i did that but ended up goign back will do it